client-js/app/elements/labrad-registry.ts

```diff
@@ -0,0 +1,195 @@
+import {RegistryApi, RegistryListing} from '../scripts/registry';
```

Comment:
Is it a bit odd that the scripts for the polymer elements aren't in the ```/scripts``` folder?